packages/front-end/components/Metrics/MetricForm/index.tsx

```diff
@@ -710,6 +759,22 @@ const MetricForm: FC<MetricFormProps> = ({
                       >
                         {value.sql ? "Edit" : "Add"} SQL <FaExternalLinkAlt />
                       </button>
+                      {value.sql != defaultSqlTemplate && (
```

Comment:
If they have edited the list of userIdTypes, this reset button will also revert those changes.  Is that clear in the UI?

Reply:
It wasn't really.  I have added a confirmation modal that explains it more.